jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -202,10 +205,12 @@ YUI.add('entity-details', function() {
 
     render: function() {
       return (
-        <div className={this._generateClasses()}>
+        <div className={this._generateClasses()}
+          ref="content"
+          tabIndex="0">
```

Comment:
I was just thinking that it may pick up the tab instead of the breadcrumb first.